adagrams/game.py

```diff
@@ -1,11 +1,108 @@
+import random
+
+    
 def draw_letters():
-    pass
+    letter_pool = {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+    # create list of strings 'letters'
+    letters = []
+    # create empty list 'hand'
+    hand = []
+    # build a random hand of 10 letters for the user
+    while len(hand) < 10:
+        letter_pool_keys = letter_pool.keys()
+        letter_list = list(letter_pool_keys)
+        if len(letter_list) > 0:
+            letter = random.choice(letter_list)
+        # Return hand that has one letter each per list item
+            if letter_pool[letter] > 0:
+                hand.append(letter)
+                # subtract from letter pool
+                letter_pool[letter] -= 1
+            elif letter_pool[letter] == 0:
+                del letter_pool[letter]
+    return hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    # create var to count  occurrences of each letter in  word
+    word = word.upper()
```

Comment:
One thing that we usually never want to do is reassign an argument that is passed inside of our function, so we would probably want to change this to something like `cap_word`.